jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/CawsSpaceProjectInfo.kt

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.caws
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.popup.ListPopup
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.wm.StatusBar
+import com.intellij.openapi.wm.StatusBarWidget
+import com.intellij.openapi.wm.StatusBarWidgetFactory
+import com.intellij.openapi.wm.impl.status.EditorBasedWidget
+import com.intellij.util.Consumer
+import java.awt.event.MouseEvent
+
+private const val WIDGET_ID = "CawsSpaceProjectInfo"
+
+class CawsStatusBarInstaller : StatusBarWidgetFactory {
+    override fun getId(): String = WIDGET_ID
+
+    override fun getDisplayName(): String = "spaceName/projectName"
```

Comment:
Can this be changed to the values for space name and project name?

Reply:
Done